### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by KamiyaGaikwad**

### CODE REVIEW

File: components/Modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    display: flex;
+    align-items: center;
+    padding: 1rem;
+    background-color: var(--card-body-border-white);
+    flex-wrap: wrap;
+    position: relative;
+    font-family: var(--font-family);
+    flex-direction: column;
+    position: fixed;
+  top: 50%;
```
Comment (mansipadgelwar) [inline components/Modal/modal.css:56]: Structure your code properly.
Rest everything looks good.
Good to merge

**Review by mansipadgelwar — APPROVED** [review]

File: components/Modal/modal.css (hunk 1 above)
Comment (KamiyaGaikwad) [reply components/Modal/modal.css:56]: Thank you for pointing it out, i will do that from next time

File: components/Modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,158 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Julius+Sans+One&display=swap');
+
+:root{
```
Comment (vinc3nati) [inline components/Modal/modal.css:4]: Everything looks good, but you can define these variables in your main file itself or rather some custom variable css file and then just import it wherever need be

**Review by vinc3nati — APPROVED** [review]

File: components/Modal/modal.css (hunk 2 above)
Comment (KamiyaGaikwad) [reply components/Modal/modal.css:4]: thank you for your suggestion, I will do that from next time

**Pull request merged**